app/views/admin/payments/delete.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l">
+    Are you sure you want to remove this payment from the payroll run?
+    </h1>
+
+    <p class="govuk-body-l">
+      <%= t("admin.payment_deletion_claim_detail",
```

Comment:
nice use of locals for this, I always forget about the one/other stuff 👍🏽

Reply:
It's super useful 👍